### DISCUSSION TITLE
How to access plugin configuration from _M.api handler

### DISCUSSION THREAD

**Discussion by khoavn** [apache/apisix#10782] [Q&A]
Hi,

For example, my plugin has this code
`function _M.api()
    return {
        {
            methods = {"GET"},
            uri = "/foo/bar",
            handler = foo_handler,
        },
    }
end`

I'm able to access this route, but don't know how to access the configurations of my plugin in `foo_handler`
Here's the schema
`
local schema = {
    type = "object",
    properties = {
        foo = {type = "string"},
        bar = {type = "string"},
    },
    required = {"foo", "bar"},
}
`

Thanks for your help

**Comment by xBoo** [comment]
anyone know?
